server/aio.c

```diff
@@ -178,6 +178,11 @@ int aio_read(
 				info(0, "child process has closed pipe");
 				break;
 
+			// There is more data available than what fit into buffer.
+			case ERROR_MORE_DATA:
+				rio->pending = 1;
+				break;
```

Comment:
Hello, I also encountered connection issues while trying rdp2tcp, and this `case ERROR_MORE_DATA` fixed them! I wanted to upstream my fixes but you were faster ;) Thanks for opening this Pull Request!

By the way, on the fix I did, I also increased the buffer size in this `case`:
```c
    min_io_size <<= 1;
    if (min_io_size > NETBUF_MAX_SIZE) min_io_size = NETBUF_MAX_SIZE;
    rio->min_io_size = min_io_size;
```

Would it make sense to add this too?